### PR TITLE
[ISSUE #50]Modify the suffix of the interface name.

### PR DESCRIPTION
**Pull request by zhangjidi2016**
## What is the purpose of the change

Modify the suffix of the interface. The query type is .query and the operation type is .do

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by vongosling — APPROVED** [review]

**Comment by vongosling** [comment]
@zhangjidi2016 It seems you have not activated your apache committer account, @duhenglucky  or @RongtongJin will help you. After that, you could merge this when at least 2 approvers got reached:-)

**Review by StyleTang — COMMENTED** [review]
`exportDlqMessage` and `batchExportDlqMessage` are read action, should add them into `role-permission.yml` by default.

File: src/main/java/org/apache/rocketmq/dashboard/controller/DlqMessageController.java
Comment (zhangjidi2016) [reply src/main/java/org/apache/rocketmq/dashboard/controller/DlqMessageController.java:86]: The exportDlqMessage uses the get request for easy export in dlqMessage.js using window.location.href
image

File: src/main/resources/role-permission.yml
Comment (StyleTang) [inline src/main/resources/role-permission.yml:37]: How about using this.
```
    - /dlqMessage/*.query
    - /dlqMessage/exportDlqMessage.do
    - /dlqMessage/batchExportDlqMessage.do
```

Because there may be write operations in this controller, e.g. `/batchResendDlqMessage.do`, ordinary users should not have write permission by default.

**Pull request merged**
